website/docs/r/secretsmanager_secret_version.html.markdown

```diff
@@ -45,6 +45,8 @@ resource "aws_secretsmanager_secret_version" "example" {
 }
 ```
 
+In Terraform 0.14 and later, use `sensitive = true` to protect the values of that `variable` from being printed in logs and console output (see ["Protect Sensitive Input Variables"](https://learn.hashicorp.com/tutorials/terraform/sensitive-variables)).
```

Comment:
A couple of small formatting suggestions here. For linking out to the Learn guide (great idea, by the way!), we don't need to wrap the text in quotes. Additionally, I think that it may be worthwhile to move this up to just below line 28, and turn it into a [callout note](https://www.terraform.io/registry/providers/docs#callouts) (by adding the `->` to the beginning of the line), so that it doesn't get missed by being below the example.

```suggestion
-> **Note:** In Terraform 0.14 and later, use `sensitive = true` to protect the values of the variable from being printed in logs and console output (see [Protect Sensitive Input Variables](https://learn.hashicorp.com/tutorials/terraform/sensitive-variables)).
```

Reply:
Ah super cool, thanks for that!